app/controllers/users_controller.rb

```diff
@@ -19,6 +20,20 @@ def update
     end
   end
 
+  def following
+    @title = "Following"
```

Comment:
move it to view and create one _form partioa which will be used in both: followin and followers views